tests/OnlineSales.Tests/SimpleTableTests.cs

```diff
@@ -73,6 +73,75 @@ public async Task CreateAndDeleteItemTest()
 
         await GetTest(testCreateItem.Item2, HttpStatusCode.NotFound);
     }
+    
+    [Fact]
+    public async Task GetTotalCountTest()
```

Comment:
@idchrono please split it into multiple tests, one for each test scenario.

Reply:
@peterliapin,
made it Theory with inline data